raptor/playback.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from __future__ import absolute_import
+
+from mozlog import get_proxy_logger
+from raptor.tools import Mitmproxy
+
+LOG = get_proxy_logger(component='playback')
+
+
+class RaptorPlayback(object):
```

Comment:
Personally I would delete this file and rename `PlaybackBase` to `Playback` (but keep the abc stuff on it). Then I'd add the `create` staticmethod idea suggested from my previous comment. So in the main raptor class you'd do:

    self.playback = Playback.create(app)  # or whatever context you need to decide which playback to create

Then the `create` method would contain the logic to pick which subclass to return. Let me know if this doesn't make sense.